arch/src/main/java/dk/ufst/arch/ReduxAndroid.kt

```diff
@@ -0,0 +1,5 @@
+package dk.ufst.arch
+
+object ReduxAndroid {
+    var debugMode = false
```

Comment:
Why is this a public mutable property? Do we want to give clients the possibility of runtime changing the `debugMode`?

Reply:
why not, maybe you just want debug output for one feature? I considered or the good old Init function which a bool parameter, but since its just setting a flag, I felt that would just complicate the usage of the lib. Might be a good for the future thought where we might perhaps allocate stuff/init other libraries :)

Reply:
Ah ok, we should document it at some point, that we explicit allow runtime to modify it. And if you want to control it per feature you will have to explicit per feature decide if it should be true or false I'm used to seeing the "_good old init function_" in other libraries and that the SIngleton pattern can give you problems, so that is why i commented on it. 
In the future we can look into controlling it per feature/`LocalStore`, maybe through `rememberLocalStore`?